src/OpenTelemetry.Exporter.Prometheus.HttpListener/Internal/PrometheusSerializerExt.cs

```diff
@@ -142,4 +167,99 @@ public static int WriteMetric(byte[] buffer, int cursor, Metric metric, Promethe
 
         return cursor;
     }
+
+    private static int WriteSumExemplar(
+        byte[] buffer,
+        int cursor,
+        in Metric metric,
+        in ReadOnlyExemplarCollection exemplarCollection)
+    {
+        var exemplars = exemplarCollection.GetEnumerator();
+        if (!exemplars.MoveNext())
+        {
+            return cursor;
+        }
+
+        ref readonly Exemplar maxExemplar = ref exemplars.Current;
+        var isLong = metric.MetricType.IsLong();
+
+        while (exemplars.MoveNext())
+        {
+            if (isLong)
+            {
+                if (exemplars.Current.LongValue >= maxExemplar.LongValue)
+                {
+                    maxExemplar = ref exemplars.Current;
+                }
+            }
+            else
+            {
+                if (exemplars.Current.DoubleValue >= maxExemplar.DoubleValue)
+                {
+                    maxExemplar = ref exemplars.Current;
+                }
+            }
+        }
+
+        return WriteExemplar(buffer, cursor, maxExemplar, metric.Name, isLong);
+    }
+
+    private static int WriteExemplar(byte[] buffer, int cursor, in Exemplar exemplar, string metricName, bool isLong)
```

Comment:
Should this go into `PrometheusSerializer.cs` alongside the rest of the byte-writing code for metrics?